Modix.Bot/Modules/QuoteModule.cs

```diff
@@ -41,6 +41,14 @@ public class QuoteModule : ModuleBase
 
                 if (message == null)
                     message = await FindMessageInUnknownChannelAsync(messageId);
+
+                var user = Context.User as IGuildUser;
+                var channel = message.Channel as ITextChannel;
+                var permissions = user.GetPermissions(channel);
+
+                if (!permissions.ViewChannel)
+                    return;
```

Comment:
Thinking it might be more user-friendly to reply with a message like "Sorry, it looks like you don't have permission to view that message." instead of just silently returning.